### ISSUE TITLE
Unable to select p-tree items when using screen reader

### ISSUE THREAD

**Issue by GeorgeDylan** [primefaces/primeng#19406]
### Describe the bug

You can not select any of the tree items when the screen reader is active using keyboard navigation. (Tested with the built in Windows 11 narrator)

This happens even on the documentation demo [https://primeng.org/tree#single](https://primeng.org/tree#single)

It allows you to navigate the tree, expand/collapse items but you cannot select an item to make it the active item.

V21

### Reason for not contributing a PR

- [x] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/tree#single

### Environment

Happens on the documentation website environment V21

### Angular version

20.3.16

### PrimeNG version

v21

### Browser(s)

Edge 144.0.3719.115 (Official build) (64-bit)

### Steps to reproduce the behavior

- Turn on screen reader
- Load page
- Attempt to select an item in the p-tree using keyboard navigation

### Expected behavior

It is expected you would be able to select the selectable menu items.
